controls/SV-251705.rb

```diff
@@ -3,28 +3,51 @@
   desc 'Without verification of the security functions, security functions may not operate correctly, and the failure may go unnoticed. Security function is defined as the hardware, software, and/or firmware of the information system responsible for enforcing the system security policy and supporting the isolation of code and data on which the protection is based. Security functionality includes, but is not limited to, establishing system accounts, configuring access authorizations (i.e., permissions, privileges), setting events to be audited, and setting intrusion detection parameters.
 
 This requirement applies to the Red Hat Enterprise Linux operating system performing security function verification/testing and/or systems and environments that require this functionality.'
-  desc 'check', 'Verify that Advanced Intrusion Detection Environment (AIDE) is installed and verifies the correct operation of all security functions.
+  desc 'check', %q(Verify that Advanced Intrusion Detection Environment (AIDE) is installed and verifies the correct operation of all security functions.
 
 Check that the AIDE package is installed with the following command:
+     $ sudo rpm -q aide
```

Comment:
Check text hasn't changed, Fix text has changed to ensure that initialization and verification steps are followed in addition to the installation of `aide`. I'd like to propose the following code after updating the default value for `file_integrity_tool` input to be an empty string:
```rb
  describe.one do
    describe package('aide') do
      it { should be_installed }
    end
    describe package(input('file_integrity_tool')) do
      it { should be_installed }
    end
  end
```

Reply:
Updated code after working with Eugene with https://github.com/mitre/redhat-enterprise-linux-7-stig-baseline/pull/189/commits/793ce63f1f4d3d45a889efe8a2b1360b167fd344

Decided to not use the following way of testing initialization because users could circumvent it:
```rb
describe "File integrity tool 'aide' should be initialized" do
  subject { file('/var/lib/aide/aide.db.gz').exist? }
  it { should be true }
end
```